gravitee-apim-console-webui/src/management/api/component/plan/api-plan-form.component.ts

```diff
@@ -86,7 +86,7 @@ type InternalPlanFormValue = {
 };
 
 type InternalPlanV3Value = Partial<PlanV3 | NewPlanV3>;
```

Comment:
suggestion(non-blocking): Maybe we should replace V3 with V2 ?
```suggestion
type InternalPlanV3Value = Partial<PlanV3 | NewPlanV3>;
```

Reply:
I agree